kitsune/questions/views.py

```diff
@@ -511,21 +492,13 @@ def aaq(request, product_key=None, category_key=None, step=1):
             product = Product.objects.get(slug=product_config["product"])
         except Product.DoesNotExist:
             raise Http404
-        else:
-            # Check if the selected product has a forum in the user's locale
-            if not product.questions_locales.filter(locale=request.LANGUAGE_CODE).count():
-                locale, path = split_path(request.path)
-                path = "/" + settings.WIKI_DEFAULT_LANGUAGE + "/" + path
-
-                old_lang = settings.LANGUAGES_DICT[request.LANGUAGE_CODE.lower()]
-                new_lang = settings.LANGUAGES_DICT[settings.WIKI_DEFAULT_LANGUAGE.lower()]
-                msg = _(
-                    "The questions forum isn't available for {product} in {old_lang}, we "
-                    "have redirected you to the {new_lang} questions forum."
-                ).format(product=product.title, old_lang=old_lang, new_lang=new_lang)
-                messages.add_message(request, messages.WARNING, msg)
-
-                return HttpResponseRedirect(path)
+        locale_has_form = product.questions_enabled(locale=request.LANGUAGE_CODE)
+        premium = product.has_subscriptions
+        request.session["aaq-context"] = {
+            "key": product_key,
+            "locale_has_form": locale_has_form,
```

Comment:
s/locale_has_form/has_public_forum to avoid confusion with the zendesk form that we are going to add

Reply:
My thinking was this would be how we enable a zendesk form per locale as well - can change the name now, and revise if it turns out that is the logic for zendesk as well though.